ui/packages/consul-ui/app/templates/dc/services/instance/exposedpaths.hbs

```diff
@@ -7,19 +7,15 @@ as |route|>
 as |item proxy|}}
   <div class="tab-section">
   {{#if (gt proxy.ServiceProxy.Expose.Paths.length 0)}}
-    <p>
-      The following list shows individual HTTP paths exposed through Envoy for external services like Prometheus. Read more about this in our <Action @href={{concat (env 'CONSUL_DOCS_URL') '/connect/registration/service-registration#expose-paths-configuration-reference'}} @external={{true}}>documentation</Action>.
-    </p>
+    {{t 'routes.dc.services.instance.exposedpaths.intro' htmlSafe=true}}
```

Comment:
Do we need to pass the `CONSUL_DOCS_URL` variable to the translations?

Reply:
Ah good question! Nah, they are all auto injected here:

https://github.com/hashicorp/consul/blob/9b22300c59c2751e08fd339fc77e1af2f2d7dcf4/ui/packages/consul-ui/app/services/i18n.js#L15-L23